deployment/cdk/opensearch-service-migration/lib/stack-composer.ts

```diff
@@ -330,7 +330,14 @@ export class StackComposer {
             this.stacks.push(openSearchStack)
             servicesYaml.target_cluster = openSearchStack.targetClusterYaml;
         } else {
-            servicesYaml.target_cluster = { endpoint: targetEndpoint, no_auth: '' }
+            servicesYaml.target_cluster = { endpoint: targetEndpoint }
+            if (fineGrainedManagerUserName) {
```

Comment:
We should probably check for both members (`fineGrainedManagerUserName`, `fineGrainedManagerUserSecretManagerKeyARN`) being present here, yeah?